pkg/estimator/server/metrics/metrics.go

```diff
@@ -15,13 +15,19 @@ const SchedulerEstimatorSubsystem = "karmada_scheduler_estimator"
 const (
 	// EstimatingTypeMaxAvailableReplicas - label of estimating type
 	EstimatingTypeMaxAvailableReplicas = "MaxAvailableReplicas"
+	// EstimatingTypeGetUnschedulableReplicas - label of estimating type
+	EstimatingTypeGetUnschedulableReplicas = "GetUnschedulableReplicas"
 )
 
 const (
 	// EstimatingStepListNodesByNodeClaim - label of estimating step
 	EstimatingStepListNodesByNodeClaim = "ListNodesByNodeClaim"
 	// EstimatingStepMaxAvailableReplicas - label of estimating step
 	EstimatingStepMaxAvailableReplicas = "MaxAvailableReplicas"
+	// EstimatingStepGetObjectFromCache - label of estimating step
+	EstimatingStepGetObjectFromCache = "GetObjectFromCache"
+	// EstimatingStepGetUnschedulablePodsOfWorkload - label of estimating step
+	EstimatingStepGetUnschedulablePodsOfWorkload = "GetWorkloadUnschedulablePods"
```

Comment:
For the metrics, does it meets [promlint](https://github.com/prometheus/client_golang/blob/f3021b0b81ae2c4adaf201b3e5de6f5f6796d433/prometheus/testutil/promlint/promlint.go#L31)?

It can be tested by E2E.

Reply:
Right, refer to https://github.com/karmada-io/karmada/pull/1030#issuecomment-984246354. They have the same struct.

Reply:
OK. 
By the way, there is also a rule for tag naming.